lib/awful/hotkeys_popup/keys/firefox.lua

```diff
@@ -7,7 +7,7 @@
 ---------------------------------------------------------------------------
 
 local hotkeys_popup = require("awful.hotkeys_popup.widget")
-local fire_rule = { class = { "Firefox" } }
+local fire_rule = { class = { "Firefox", "firefox" } }
```

Comment:
@psychon 
Would matching by name be any better?

```suggestion
local fire_rule = { name =  "Firefox" }
```

Reply:
I don't think so, no. Window name changes much more easily (and e.g. usually contains the name of the web page).

Feel free to just ignore me here. I am merely confused...